docs/en/quick-answers/troubleshooting/module-tank-has-no-attribute-support-url.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: default
+title: "Error module 'tank' has no attribute 'support_url'" when launching {% include product %} Desktop
+pagename: module-tank-has-no-attribute-support-url
+lang: en
+---
+
+# "Error module 'tank' has no attribute 'support_url'" when launching {% include product %} Desktop
+
+## Issue
+
+The following message appears when launching {% include product %} Desktop after upgrading the version:
+
+```
+{% include product %} Desktop Error:
+Error: module 'tank' has no attribute 'support_url'
+```
+
+## Causes
+
+'support_url' was introduced in tk-core v0.19.18. and the Descriptor version is incompatible with the newer {% include product %} Desktop version 1.7.3.
```

Comment:
I think that the "'support_url' was introduced in tk-core v0.19.18." part is more an explanation than the cause itself and that's why I believe it should be displayed after

Like this: 
The Descriptor version is incompatible with the newer {% include product %} Desktop version 1.7.3.
'support_url' was introduced in tk-core v0.19.18.

Reply:
Fixed, thanks @dsauve-adsk !